package.json

```diff
@@ -134,6 +135,21 @@
           "type": "boolean",
           "default": true,
           "description": "Trigger suggest on both insertion AND replacement of new character inside a wiki-link. Defaults true. Set false to only trigger suggest on insertion. See PR #69 for details."
+        },
+        "vscodeMarkdownNotes.showFileExtensionInPreview": {
```

Comment:
Let's rename this to `previewShowFileExtension` to sort of 'namespace' with 'preview'

Reply:
:+1: